lib/lte_link_control/lte_lc.c

```diff
@@ -493,49 +496,37 @@ static void at_handler(void *context, const char *response)
 static int enable_notifications(void)
 {
 	int err;
-	char buf_sub[35];
 
 	/* +CEREG notifications, level 5 */
-	err = at_cmd_write(cereg_5_subscribe, NULL, 0, NULL);
+	err = nrf_modem_at_printf(AT_CEREG_5);
 	if (err) {
-		LOG_ERR("Failed to subscribe to CEREG notifications");
+		LOG_ERR("Failed to subscribe to CEREG notifications, error: %d", err);
 		return err;
```

Comment:
return -EFAULT ?

Reply:
I landed on keeping the error codes as is in the internal functions, so this is on purpose